react/features/dialog/actionTypes.js

```diff
@@ -0,0 +1,20 @@
+import { Symbol } from '../base/react';
+
+/**
+ * Action to hide the dialog.
+ *
+ * {
+ *     type: HIDE_DIALOG,
+ * }
+ */
+export const HIDE_DIALOG = Symbol('HIDE_DIALOG');
+
+/**
+ * Action to display the dialog.
+ *
+ * {
+ *     dialogType: string
+ *     type: SHOW_DIALOG,
```

Comment:
You had the good game going before. A dialog may need options, this action should take them. These options might be dynamic, hence the need to pass then when we show it, instead of defining them statically.